libsolidity/interface/StandardCompiler.h

```diff
@@ -55,6 +55,7 @@ class StandardCompiler: boost::noncopyable
 	Json::Value compileInternal(Json::Value const& _input);
 
 	CompilerStack m_compilerStack;
+	ReadFile::Callback const& m_readFile;
```

Comment:
This cannot be a `const&`, because it might be assigned from a temporary (for example in the case the `_readFile` constructor argument is missing) and thus the readfile object might be destroyed before the StandardCompiler.